app/javascript/packs/intl-tel-input.js

```diff
@@ -6,8 +6,8 @@ const intlCode = document.querySelector('#new_phone_form_international_code');
 
 // initialise plugin
 intlTelInput(telInput, {
-  preferredCountries: ['us', 'ca'],
-  excludeCountries: ['io', 'ki', 'nf', 'nr', 'nu', 'sh', 'sx', 'tk', 'wf'],
```

Comment:
As of https://github.com/18F/identity-idp/pull/4334, `KI` is now a supported country so it makes for a good test case